getting_started_with_acceptance_tests.md

```diff
@@ -0,0 +1,80 @@
+# A Larger Product and Team
+At Ubiqua, as we started to grow both our product and engineering team, we realized some things had to be re-prioritezed and fixed. Some of the problems we were having were:
+
+- The onboarding of a single person to our software engineering team took more time than we desired. This was mainly being cause due to legacy code and the bad documentation of features and processes.
+- Software engineers were not able to determine if a specific defect fix or feature broke older functionality.
+- Slow production line. We needed to spend a lot of time manually testing if new functionality behaved as expected and did not break anything else.
+
+As a result of this, we decided we would start off by changing our development workflow to a more collaborative and documented process, where changes to the codebase followed a specific set of actions. These actions are:
+  - All new features must include tests (bonus points for using TDD).
+  - Bug fixes must include a corresponding regression test.
+  - One test must be added to the existing codebase every week.
+
+> You can read more about them in this blog post: [Refactoring Legacy Code](https://github.com/ubiqua/engblog/blob/master/refactoring_legacy_code.md).
+
+One of the tools we chose for our plan of action is [Cucumber](https://cucumber.io/), and in this blogpost I will guide you through the basics and good practices of writing acceptance tests with it.
```

Comment:
"we chose.. was" keep the same tense. Also, consider splitting into two sentences, instead of using ", and"